packages/manager/src/queries/cloudpulse/services.ts

```diff
@@ -28,7 +28,6 @@ export const useCloudPulseJWEtokenQuery = (
   return useQuery<JWEToken, APIError[]>({
     ...queryFactory.token(serviceType, request),
     enabled: runQuery,
-    placeholderData: keepPreviousData,
```

Comment:
if we have the previousData, why it is wrong, since we drive the entire logic now based of isJweTokenFetching